tst/EnergyPlus/unit/ElectricPowerServiceManager.unit.cc

```diff
@@ -1230,3 +1230,148 @@ TEST_F(EnergyPlusFixture, Battery_checkUserEfficiencyInputTest)
     EXPECT_NEAR(functionResult, expectedResult, 0.00001);
     EXPECT_FALSE(errorsFound);
 }
+
+TEST_F(EnergyPlusFixture, Battery_checkChargeDischargeVoltageCurves)
+{
+    std::string battName;
```

Comment:
battName doesn't look like it's used.

Reply:
I'll get rid of it.  Forgot that it was there when I just assigned the name in-line.

Also, just for clarity (not sure why it's not letting me comment on this above), the reworded error message--sounds good to me, but are you advocating for only doing the check every tenth or only reporting on the tenths when there is a problem?

Reply:
I think the high resolution checks are good but if there is a problem report out the low resolution results. The original message was too bulky. You may have other ideas of how to briefly report out the issue.

Reply:
Commit made, merged in develop, and pushed this all up to GitHub.  Let me know if there is anything else that needs to be addressed.  Thanks!